python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -365,7 +369,7 @@ def kill(self):
         if not self.id:
             return None
         dirac_cmd = 'kill(%d)' % self.id
-        result = execute(dirac_cmd)
+        result = execute(dirac_cmd, return_raw_dict=True)
```

Comment:
Almost all calls within `GangaDirac` itself yes turn this off as they have code explicitly written to deal with this and I'm wanting to make _minimal_ changes elsewhere in the code. Touching things breaks things and I've just managed to get this stable and reasonably performant.

As I've said we cab change policy about if we want to explicitly request this, but imo it makes more sense to have this turned on as people don't think to do error checking and to pick up the cases where people are using Ganga to talk to DIRAC (aka LHCb expert users!).

This does however break backwards compatibility and there was a desire to have a `GangaDiracException` thrown from this level. If this needs to change I'd like to see some discussion on #786 about whether we infact want `GangaDiracException`.